docs/software/admin.md

```diff
@@ -638,6 +639,7 @@ The output will look something like
             "disagree" : 0,
             "fail_at" : 2,
             "hash" : "d5c247",
+            "lag_ms" : 430,
```

Comment:
you forgot to update the output of the `quorum` in the section `Quorum set diagnostics`

Reply:
ah yes, grepped for the wrong thing. Fixed!